project/app/main.py

```diff
@@ -5,8 +5,8 @@
 from app.api import predict, viz
 
 app = FastAPI(
-    title='YOUR-PRODUCT-TEAM-NAME DS API',
-    description='Replace this placeholder text',
+    title='saverlife-a',
+    description='end point for ds/web',
```

Comment:
For description: 

"DS API containing endpoints that provide visualizations and predictions (in JSON)."

Thought???